phpcs.xml

```diff
@@ -6,6 +6,10 @@
 
     <config name="testVersion" value="5.4-"/>
 
+    <arg value="s"/>
```

Comment:
I renamed this so that `phpcs` in Travis will pick this up, without having to pass it as an argument, like `phpcs --standard=phpcs.ruleset.xml`

Reply:
If this name is important, we could keep the name and just pass it as an argument in `.travis.yml`